### PR TITLE
adding trim, replaceregex, replacall, indexany operations

### PR DESCRIPTION
**Pull request by abramvandergeest**

### CODE REVIEW

File: operations/image_processing/img2tensor/operation.go [hunk 1]
```diff
@@ -107,11 +108,13 @@ func (a *Operation) Eval(inputs map[string]interface{}) (interface{}, error) {
 		}
 		a.logger.Info("Operation img2tensor Completed.")
 		a.logger.Debug("Values of first pixel ", img[0][0][0])
+		fmt.Println(img[0][0])
```
Comment (skothari-tibco) [inline operations/image_processing/img2tensor/operation.go]: Is this needed ?

File: operations/image_processing/img2tensor/operation.go [hunk 2]
```diff
@@ -107,11 +108,13 @@ func (a *Operation) Eval(inputs map[string]interface{}) (interface{}, error) {
...
 		return img, nil
 
 	}
 	a.logger.Info("Operation img2tensor Completed.")
 	a.logger.Debug("Values of first pixel ", singleimg[0][0])
+	fmt.Println(singleimg[0])
```
Comment (skothari-tibco) [inline operations/image_processing/img2tensor/operation.go]: This too?

File: operations/string_processing/indexany/operation.go [hunk 3]
```diff
@@ -0,0 +1,37 @@
+package lastindex
```
Comment (skothari-tibco) [inline operations/string_processing/indexany/operation.go]: Is this package name correct in indexany?

File: operations/image_processing/img2tensor/operation.go (hunk 1 above)
Comment (abramvandergeest) [reply operations/image_processing/img2tensor/operation.go]: removed

File: operations/image_processing/img2tensor/operation.go (hunk 2 above)
Comment (abramvandergeest) [reply operations/image_processing/img2tensor/operation.go]: removed

File: operations/string_processing/indexany/operation.go (hunk 3 above)
Comment (abramvandergeest) [reply operations/string_processing/indexany/operation.go]: changed

**Review by skothari-tibco — APPROVED** [review]

**Pull request merged**